src/SFA.DAS.FindApprenticeship.Jobs/Infrastructure/Api/Requests/GetLiveVacancyApiRequest.cs

```diff
@@ -1,15 +1,14 @@
-﻿using System;
-using SFA.DAS.FindApprenticeship.Jobs.Domain.Interfaces;
+﻿using SFA.DAS.FindApprenticeship.Jobs.Domain.Interfaces;
 
 namespace SFA.DAS.FindApprenticeship.Jobs.Infrastructure.Api.Requests;
 public class GetLiveVacancyApiRequest : IGetApiRequest
 {
-    private readonly Guid _vacancyId;
+    private readonly long _vacancyReference;
 
-    public GetLiveVacancyApiRequest(Guid vacancyId)
+    public GetLiveVacancyApiRequest(long vacancyReference)
     {
-        _vacancyId = vacancyId;
+        _vacancyReference = vacancyReference;
     }
 
-    public string GetUrl => $"livevacancy?vacancyId={_vacancyId}";
-}
+    public string GetUrl => $"livevacancy?vacancyRef={_vacancyReference}";
```

Comment:
It might be nicer if we made this a little more restful, eg `livevacancies/{vacancyReference}` so that it is similar to the getall endpoint